keyboards/handwired/maverick0197/keydeck8/info.json

```
@@ -0,0 +1,39 @@
+{
+	"keyboard_name": "keydeck8",
+	"manufacturer": "Maverick0197",
+	"maintainer": "Maverick0197",
+	"bootloader": "caterina",
```

This sets processor and bootloader:
```suggestion
    "development_board": "promicro",
```